core/state_transition.go

```diff
@@ -526,7 +571,16 @@ func (st *StateTransition) refundGas(refundQuotient uint64) {
 
 	// Return ETH for remaining gas, exchanged at the original rate.
```

Comment:
In EVM there is refund mechanism that if evm storage setting from 1 to 0, the gas will refund to `msg.sender`.
This refund gas will be distributed to `msg.sender` and `sponsor` now.